From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: "Jan Alexander Steffens (heftig)" <heftig@archlinux.org>
Date: Thu, 20 Oct 2022 20:03:36 +0000
Subject: [PATCH] pam-arch: Remove user_readenv=1 from pam_env

The insecure `user_readenv` setting has been deprecated with pam 1.5.0
and will be removed in a future release.
---
 data/pam-arch/gdm-launch-environment.pam | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/data/pam-arch/gdm-launch-environment.pam b/data/pam-arch/gdm-launch-environment.pam
index 20d1810a68f5..3c4ad407df89 100644
--- a/data/pam-arch/gdm-launch-environment.pam
+++ b/data/pam-arch/gdm-launch-environment.pam
@@ -14,4 +14,4 @@ session    optional                    pam_keyinit.so       force revoke
 session    required                    pam_succeed_if.so    audit quiet_success user in gdm:gnome-initial-setup
 session    optional                    pam_permit.so
 -session   optional                    pam_systemd.so
-session    required                    pam_env.so           user_readenv=1
+session    required                    pam_env.so
